broker-core/src/test/java/io/zeebe/broker/logstreams/processor/KeyGeneratorTest.java

```diff
@@ -74,10 +74,24 @@ public void shouldGenerateNextKetWithoutStateController() {
     assertThat(key).isEqualTo(0);
   }
 
+  @Test
```

Comment:
We should have tests that explicitly test the key format, because this is now essentially API.